lib/sles4sap.pm

```diff
@@ -645,6 +646,7 @@ sub check_replication_state {
     # Wait by default for 5 minutes
     my $time_to_wait = 300;
     my $cmd = "su - $sapadm -c 'python2 exe/python_support/systemReplicationStatus.py'";
```

Comment:
I wouldn't use the full path to the script but run the `cdpy` command instead like it's done here:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/hana_test.pm#L28

In the [test_python3()](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/hana_test.pm#L16) function above we check that the scripts compile for Python 3 on HANA >= 2.00.060:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/hana_test.pm#L73